fastchain/document/base.py

```diff
@@ -39,7 +39,7 @@ class Document(BaseDoc):
     )
     doc_summary: Optional[str] = "NA"
     pages: Optional[DocList[Page]]
-    chunks: Optional[DocList[Chunk]]
+    chunks: DocList[Chunk]
```

Comment:
Shouldn't it be optional only as we discussed yesterday?
```suggestion
    chunks: Optional[DocList[Chunk]]
```